jenkins/Dockerfile-blossom.ubuntu

```diff
@@ -25,7 +25,7 @@
 #       UCX_VER=1.13.1
 ###
 
-ARG CUDA_VER=11.0.3
```

Comment:
BTW https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=11.0.3 is still supported, I guess we may not update this one as we still trying to use the minimal cuda version image in our nightly and premerge.

Reply:
revert the change, thanks